meerkat_frontend/src/js/technical/tableManager.js

```diff
@@ -439,6 +461,7 @@ function drawAlertsTable(containerID, alerts, variables) {
             sortable: true
         }];
 
+        var columns = columns1.concat(columns2);
```

Comment:
Rather than this columns1/2 contacatonation, with a third push function - I think it would be simpler and clearer to just use the javascript splice method.  1) define the complete original columns array as it was previously 2) use the splice function in an optional second step to insert the district column at the right index in the columns array.